src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java

```diff
@@ -53,7 +51,7 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
     {
         if(args.length < 2)
         {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 8, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)